modules/data-entry/src/main/frontend/src/questionnaire/NumberQuestion.jsx

```diff
@@ -159,15 +152,51 @@ function NumberQuestion(props) {
 
   let hasAnswerOptions = !!(props.defaults || Object.values(props.questionDefinition).some(value => value['jcr:primaryType'] == 'cards:AnswerOption'));
 
+  // Generate message about accepted min/maxValues
+  let minMaxMessage = "";
+  if (typeof minValue !== "undefined" || typeof maxValue !== "undefined") {
+    minMaxMessage = "Please enter values ";
+    if (typeof minValue !== "undefined" && typeof maxValue !== "undefined") {
+      minMaxMessage = `${minMaxMessage} between ${minValue} and ${maxValue}`;
+    } else if (typeof minValue !== "undefined") {
+      minMaxMessage = `${minMaxMessage} of at least ${minValue}`;
+    } else {
+      minMaxMessage = `${minMaxMessage} of at most ${maxValue}`;
+    }
+    if (hasAnswerOptions) {
+      minMaxMessage = `${minMaxMessage} or select one of the options`;
+    }
+  }
+
+
   return (
     <Question
       disableInstructions={hasAnswerOptions}
       currentAnswers={ typeof(input) != "undefined" && input != "" ? 1 : 0 }
       {...props}
       >
-      {error && <Typography color='error'>{errorText}</Typography>}
+      { minMaxMessage &&
+        <Typography
+          component="p"
+          color={minMaxError ? 'error' : 'textSecondary'}
+          className={classes.answerInstructions}
+          variant="caption"
+        >
+          { minMaxMessage }
+        </Typography>
+      }
       { isRange ?
         <>
+        { rangeError &&
+          <Typography
+            component="p"
+            color="secondary"
```

Comment:
Why is the first error red but the second one orange? In other words, should we do this?

```suggestion
            color="error"
```

Reply:
Addressed in ffb64ca .